packages/components/src/card/src/Card.css

```diff
@@ -118,7 +118,7 @@
     padding: var(--o-ui-sp-6);
 }
 
-.o-ui-card-horizontal > .o-ui-card-illustration+.o-ui-card-aside {
+.o-ui-card-horizontal > .o-ui-card-illustration + .o-ui-card-aside {
```

Comment:
@fraincs the spacing around these operator is inconsistent. Is this the way you would like to write them, or without the spaces?

Reply:
This is what happens on save on my end, been fighting with them a lot ` + ` is perfect.